openedx/core/djangoapps/site_configuration/helpers.py

```diff
@@ -29,6 +29,14 @@ def get_current_site_configuration():
         return None
 
 
+def get_current_site_configuration_values(default={}):
```

Comment:
@shimulch, mutable default values are bad. Can you use `None` and in the first line of the function use something like

```python
default = default if default is not None else {}
```

Reply:
@lgp171188, thanks for the suggestion. :-)